### PR TITLE
Mssql dialect support

### PR DESCRIPTION
**Pull request by guruofgentoo**
Also sets up AppVeyor

### CODE REVIEW

**Comment by nZac** [comment]
@guruofgentoo thanks for sending this over... sorry it has taken me a long time to get to.

I just get AppVeyour setup in master. Can you get this branch synced up?

**Comment by guruofgentoo** [comment]
Rebased on latest master changes, using the AppVeyor config that was on this branch. The necessary wheels are already there, and everything passes.

File: keg_apps/db/user-config-tpl-appveyor.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+class TestProfile(object):
```
Comment (nZac) [inline keg_apps/db/user-config-tpl-appveyor.py]: Is this and the `cp` command in appveyour.yaml necessary? Can't we just use the default profile.

File: keg/tests/test_templating.py [hunk 2]
```diff
@@ -32,13 +34,17 @@ def test_include(self):
         assert len(self.assets.content['js']) == 1
         asset_name, filepath, content = self.assets.content['js'].pop()
         assert asset_name == 'assets_in_template.js'
-        assert filepath.endswith('keg_apps/templating/templates/assets_in_template.js')
+        assert filepath.endswith(
+            os.path.join('keg_apps', 'templating', 'templates', 'assets_in_template.js')
```
Comment (nZac) [inline keg/tests/test_templating.py]: 😍  Thanks for fixing this.

**Pull request merged**